docs/book/component-guide/deployers/local.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Deploying pipelines on your local machine as background processes.
+---
+
+# Local Deployer
+
+The local deployer is a [deployer](./) flavor that comes built-in with ZenML and deploys pipelines on your local machine as background processes.
```

Comment:
will this URL work? safer to specify `./README.md`?

Reply:
this works just fine, I checked. It's also used similarly in other parts of the docs